corpus/filter.py

```diff
@@ -111,14 +109,20 @@ def run(self):
 
 
 class FilterSegmentsByAlignmentConfidenceJob(Job):
-    def __init__(self, alignment_logs, percentile, crp=None, plot=True, absolute_threshold=None):
+    def __init__(
+        self,
+        alignment_logs: Dict[int, Path],
+        percentile: float,
+        absolute_threshold: Optional[float] = None,
```

Comment:
I wondered why the apptek-tests failed, but you changed the order here which could be a potential cause. This is not a good idea unless there was a `*` before, enforcing only named parameters.

Reply:
please disregard the apptek test for the next few minutes, this is a problem on our side that should be fixed soon.

Reply:
Oh right, I changed it to match the order in the docstring but it's certainly a better idea to do it the other way around.

Reply:
> please disregard the apptek test for the next few minutes

the problem was resolved and the test passes now. pls continue as you were :)